pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -375,7 +375,7 @@ func (k *Deployer) renderManifests(ctx context.Context, _ io.Writer, builds []gr
 	if err != nil {
```

Comment:
Can you add some tracing for `Deploy`, `Render`, and `Apply` as you've done for the kubectl deployer?

Reply:
Done